proto/wechaty/puppet/friendship.proto

```diff
@@ -62,8 +62,11 @@ message FriendshipSearchWeixinResponse {
 
 message FriendshipAddRequest {
   string contact_id = 1;
-  string hello = 2;
+  google.protobuf.StringValue hello = 2;
+  google.protobuf.StringValue inviter_room_id = 3;
```

Comment:
I feel that the `source_room_id` will be more intuition than `inviter_room_id`, how do you guys think?

Reply:
Agree

Reply:
Agree, and how about rename `inviter_contact_id` as `recommendContactId`? Or `sourceContactId`?

Reply:
I'd like to prefer `sourceContactId`

Reply:
Should be `source_contact_id` in grpc world.

Reply:
OK